posts/views.py

```diff
@@ -83,7 +85,7 @@ def get_post(self, pk):
     def get(self, request, pk):
         post = self.get_post(pk)
         serializer = PostSerializer(post)
-        return Response(serializer.data)
+        return render(request, 'post/post.html', context={'post': serializer.data})
```

Comment:
This is in the user view not the post view. Also make it so that the new views do no conflict with the existing api views. The api ones should be accessable at /api/<view> and this front end should be at /posts/<id> or whatever the spec says.

Reply:
We can make the API endpoints respond with json or HTML depending on the content the user specifies. This might still conflict with the specs though

Reply:
Good catch. Forgot about the spec for a bit. The spec just says that the calls to the service/pos should return json so we can do /frontend/posts/. I think this is super easy combared to using the html renders because I had a massive issue with it when doing the approving users.